credsweeper/deep_scanner/deb_scanner.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from abc import ABC
+from typing import List, Optional
+
+from credsweeper.common.constants import ASCII, MIN_DATA_LEN
+from credsweeper.credentials import Candidate
+from credsweeper.deep_scanner.abstract_scanner import AbstractScanner
+from credsweeper.file_handler.data_content_provider import DataContentProvider
+
+logger = logging.getLogger(__name__)
+
+
+class DebScanner(AbstractScanner, ABC):
+    """Implements deb (ar) scanning"""
+
+    def data_scan(
+            self,  #
```

Comment:
Please remove blank comments from code
```suggestion
            self,
```

Reply:
unfortunately, the sign is necessary to keep formatting yapf suitable